include/crow/app.h

```diff
@@ -120,7 +120,7 @@ namespace crow
             return *this;
         }
 
-        ///Set the server name (default Crow/0.3)
+        ///Set the server name (default Crow/master)
```

Comment:
I think that you can remove info about default value. Or change to something like `Crow/<VERSION>`

Reply:
My bad, I just changed every `0.3` to master without looking much.